tasks/func.py

```diff
@@ -130,6 +130,16 @@ def invoke(ctx, user, func, input_data=None, mpi=None, graph=False):
     print("Success:\n{}".format(response.text))
 
 
+@task
+def update(ctx, user, func, clean=False, debug=False, native=False):
+    """
+    Combined compile and upload
+    """
+    compile(ctx, user, func, clean=clean)
```

Comment:
I can not remember the flushing policy we agreed on, but could you double-check that running update twice with the same `user/func` with different function code yields the expected result?

Reply:
Ah yes, good point, this should really end with a flush otherwise you'll still get the old one hanging around.